packages/react-icons-mdl2-branded/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@fluentui/react-icons-mdl2-branded",
+  "version": "1.0.0-beta.0",
+  "description": "Branded SVG icons from the MDL2 icon set",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "Microsoft Fabric Assets License",
```

Comment:
I looked into this a bit and per the [npm docs](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#license) apparently this field is expected to either be one of a known list of license, or possibly conform to a certain format for custom licenses: `SEE LICENSE IN <filename>` which here would be `SEE LICENSE IN LICENSE`...seems weird, plus it's not actually clear whether that format is prescriptive or just a suggestion.

Reply:
I think as weird as it sounds, I'll change it to `SEE LICENSE IN LICENSE`, if it is on the npm docs site